packages/api/src/migrations/0000-create-destination-and-code-tables.ts

```diff
@@ -0,0 +1,12 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20220307172403 extends Migration {
+  async up(): Promise<void> {
+    this.addSql('create table "code" ("iata" varchar(255) not null, "city" text not null, "country" text not null);');
+    this.addSql('alter table "code" add constraint "code_pkey" primary key ("iata");');
+
+    this.addSql('create table "destination" ("id" bigserial primary key, "createdAt" timestamptz(0) not null default clock_timestamp(), "updatedAt" timestamptz(0) not null default clock_timestamp(), "origin" text not null, "miles" int4 not null, "period" text not null, "destination" varchar(255) not null);');
+
+    this.addSql('alter table "destination" add constraint "destination_destination_foreign" foreign key ("destination") references "code" ("iata") on update cascade;');
+  }
```

Comment:
be sure to manually add the `down` portion of the migration as well! It should be the inverse of what is in the up (in this case you would want to alter the table to remove the changes and then drop the table for destination first and then code)
```suggestion
  }
  
    async down(): Promise<void> {
    //inverse of up goes here
  }
```